tests/components/onewire/test_entity_sysbus.py

```diff
@@ -1,5 +1,5 @@
 """Tests for 1-Wire devices connected on SysBus."""
-from unittest.mock import PropertyMock, patch
+from unittest.mock import patch
```

Comment:
Updating this to from tests.async_mock import patch` would improve it.

Reply:
done in ccd9314